pull_request_template.md

```diff
@@ -0,0 +1,16 @@
+What
+----
+
+***What you have changed: ***
+
+How to review
+-------------
+
+***How someone else can review your changes: ***
+
+Checklist
+---------
+
+- [ ] Run ./pre-commit.sh
+- [ ] Updated any necessary documentation
+
```

Comment:
Can we add?
[ ] Thought about potential vulnerabilities

Reply:
Can we also add?
[ ] Added some comments to this PR about how to test/verify this change